gallery/how_to/work_with_relay/using_with_pipeline_executor.py

```diff
@@ -0,0 +1,262 @@
+###########################################################################
+# Set the compile target of the second subgraph module for example as LLVM.
+pipe_config[mod0].target = "llvm"
+pipe_config[mod0].dev = tvm.cpu(0)
+###############################################################################
+# Set the cpu afinity for control flow, for example using cpu 0 for control flow.
+pipe_config[mod1].cpu_affinity = "0"
+pipe_config[mod1].export_cc = None
+##############################################################
+# Set the compile target of the second subgraph module as LLVM.
+pipe_config[mod1].target = "cuda"
+pipe_config[mod1].dev = tvm.device("cuda", 0)
+pipe_config[mod1].build_func = cutlass_build
+pipe_config[mod1].export_cc = "nvcc"
+#################################################################################
+# Set the cpu afinity for control flow, for example using cpu 1 for control flow.
+pipe_config[mod1].cpu_affinity = "1"
+pipe_config["input"]["data"].connect(pipe_config[mod0]["input"]["data"])
+pipe_config[mod0]["output"][0].connect(pipe_config[mod1]["input"]["data_n_0"])
+pipe_config[mod1]["output"]["0"].connect(pipe_config["output"][0])
+######################################
+# The pipeline configuration as below.
+"""
+print(pipe_config)
+ Inputs
+  |data: mod0:data
+
+ output
+  |output(0) : mod1.output(0)
+
+ connections
+  |mod0.output(0)-> mod1.data_n_0
+"""
+
+# sphinx_gallery_start_ignore
+from tvm import testing
+
+# testing.utils.install_request_hook(depth=3)
+# sphinx_gallery_end_ignore
+##############################
+# Build the pipeline executor.
+# ----------------------------
+with tvm.transform.PassContext(opt_level=3):
+    pipeline_mod_factory = pipeline_executor_build.build(pipe_config)
+###############################################
+# Export the parameter configuration to a file.
+directory_path = tvm.contrib.utils.tempdir().temp_dir
+#############################################
+# If the directory does not exist, create it.
+if not os.path.exists(directory_path):
+    os.makedirs(directory_path)
+config_file_name = pipeline_mod_factory.export_library(directory_path)
+################################################################
+# Use the load function to create and initialize PipelineModule.
+# --------------------------------------------------------------
+pipeline_module = pipeline_executor.PipelineModule.load_library(config_file_name)
+
+############################
+# Run the pipeline executor.
+# --------------------------
+# Allocated a input data.
+data = np.random.uniform(-1, 1, size=data_shape).astype("float16")
+pipeline_module.set_input("data", tvm.nd.array(data))
+##########################################################################
+# Run the two subgraph in pipeline mode and get the output asynchronously.
+pipeline_module.run()
+outputs = []
+while not outputs:
+    outputs = pipeline_module.get_output()
+    time.sleep(0.001)
```

Comment:
why need sleep?

Reply:
when we use the "while loop" to poll the output data, the backend in the pipeline may still in the state of running, because the execution of the pipeline executor is in a  "asynchronous" mode, hence we need use sleep to reduce the
usage of cpu which may is used by a backend.

Reply:
removed sleep